s2i-gows/README.md

```diff
@@ -4,12 +4,15 @@
 
 ## Quickstart
 
-Here's a simple on-liner to get you started.
+1. Clone this repository:
+   `git clone https://github.com/redhat-cop/containers-quickstarts`
+2. Run `ansible-galaxy install -r requirements.yml --roles-path=roles`
+3. `cd containers-quickstarts/s2i-gows`
```

Comment:
2 and 3 need to be swapped.

Reply:
haha, yeah that would help, updated.
Thank you @sabre1041